spec/models/category_spec.rb

```diff
@@ -5,28 +5,29 @@
   it { should validate_presence_of(:name)}
 
   describe "recent videos" do 
+    before do 
+      @drama = Category.create(name: "Drama")
+    end 
+    let(:subject) { @drama.recent_videos }
+
     it "should show only 6 videos if more than 6 videos exists in a category" do 
-      thriller = Category.create(name: "thriller")
       7.times do 
-        video = Video.create(title:"24", description: "Fast thriller", category_id: thriller.id)
+        Fabricate(:video, category_id: @drama.id)
       end
-      expect(thriller.recent_videos.count).to be(6)
+      expect(subject.count).to eq(6)
     end
 
     it "should show all videos if less than 6 videos exists in a category" do
-      comedy = Category.create(name: "comedy")
       4.times do 
-        video = Video.create(title: "Family Guy", description: "Funny show", category_id: comedy.id)
+        Fabricate(:video, category_id: @drama.id)
       end
-      expect(comedy.recent_videos.count).to eq(comedy.videos.count)
+      expect(subject.count).to eq(@drama.videos.count)
     end
 
     it "should show all videos order by created_at" do 
```

Comment:
`should` in example spec message looks redundant - not bringing anything to the description - I recommend starting with verb in 3rd person singular - read more here - http://www.betterspecs.org/#should